st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -148,11 +148,13 @@ def run(self, action_parameters):
         stdout, stderr = process.communicate()
         exit_code = process.returncode
 
-        if ACTION_OUTPUT_RESULT_DELIMITER in stdout:
-            split = stdout.split(ACTION_OUTPUT_RESULT_DELIMITER)
+        stream_to_process = stderr if exit_code else stdout
```

Comment:
Discussed offline. For now, either the action returns a result or an exception. So this is not needed.